src/Components/Desktop.jsx

```diff
@@ -1,13 +1,18 @@
+import { isMobile } from "react-device-detect";
```

Comment:
Have you considered using viewport size/aspect ratio to determine is "mobile" or not? This should fix the problem with iPad too since the biggest issue is rendering a vertical version (windows open, scrolling, no drag, no icons etc) of the site and a horizontal version (desktop experience). Perhaps calling it "mobile" and not "vertical" in the design doc was a mistake since it's more about the orientation of the page than the device used. 
Using viewport will also mean if someone rotates their phone, they view the desktop experience.